runtime/strings/bytes.cpp

```diff
@@ -182,6 +182,27 @@ SortBytes hook_BYTES_replaceAt(SortBytes b, SortInt start, SortBytes b2) {
   return b;
 }
 
+SortBytes
+hook_BYTES_memset(SortBytes b, SortInt start, SortInt count, SortInt value) {
+  uint64_t ustart = get_ui(start);
+  uint64_t ucount = get_ui(count);
+  uint64_t uend = ustart + ucount;
```

Comment:
Perhaps implement this in terms of `hook_INT_add`? It's a _really_ unlikely corner case, but it would mean we get an overflow error message rather than just a silent failure.

Reply:
I don't think it makes sense to use arbitrary precision arithmetic here, but we should  write a check for overflow since it would be undefined behavior. We can just throw an exception if the addition would overflow.

Reply:
Yeah, that's all I'm getting at - my thought was you'd catch the overflow when lowering the result of the MPZ addition back to a regular integer.